twake/backend/node/src/services/workspaces/web/controllers/workspaces.ts

```diff
@@ -191,7 +191,7 @@ export class WorkspacesCrudController
 
     const workspaceUserRole = await this.getWorkspaceUserRole(request.params.id, context);
 
-    if (workspaceUserRole !== "admin") {
+    if (workspaceUserRole !== "moderator") {
```

Comment:
Could we replace this by the utils function "hasWorkspaceAdminLevel()" ?

Reply:
done